bin/build-constraints.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eu
+
+runFromHook=$1
+[[ $runFromHook == "true" ]] && cd ..
+
+. ./bin/common.sh
+
+gatekeeperArtifactsPath="./values/gatekeeper/constraints"
+function build() {
+  echo "Building constraints artifacts from stack policies."
+  local policiesPath="./policies"
+  rm -f "$gatekeeperArtifactsPath/*"
+  konstraint create $policiesPath -o $gatekeeperArtifactsPath
+}
+
+function decorate() {
+  echo "Decorating constraints files with parameters."
+  for constraint in $(hf_values | yq r -j - 'charts.gatekeeper' | jq --raw-output -S -c '.constraints[]  | {(.policyName):.parameters}'); do
+    local key=$(echo $constraint | jq '. | keys[0]' | sed 's/"//g')
```

Comment:
By using `jq --raw-output` you can get rid of `sed`
e.g.:

local key=$(echo $constraint | jq --raw-output '. | keys[0]')

Reply:
ok, nice catch!